### ISSUE TITLE
Consistency for static code analysis support

### ISSUE THREAD

**Issue by peter279k** [antidot-framework/container#22]
**Is your feature request related to a problem? Please describe.**

- Add static code analysis support and consistency.

**Describe the solution you'd like**

- It seems that this package has been defined with `phpstan` installation. However, other packages on this org are defined with `Psalm` installation. I think we should determine one of static code analysis tools support because of consistency.

**Describe alternatives you've considered**

N/A

**Additional context**

N/A

**Comment by kpicaza** [comment]
Hi @peter279k, sorry for the delay. I think the reason is that I start using psalm time after creating this package. You are right, it should have Psalm in the pipeline for consistency.  
I like to use both Phpstan and Psalm, both do the same but they do in a different way, and both detect some different issues. In Pheature-Flags we are using this config and it works pretty well. https://github.com/pheature-flags/toggle-model/blob/1.0.x/composer.json (look at scripts section).

Thank you very much, I have to find some time to update Antidot packages' pipeline and PHP 8.1 compatibilities.

**Comment by peter279k** [comment]
Hi @kpicaza, thanks for your reply and clear explanation.

Due to your explanation, I think the current solution should let Psalm and PHPStan running on every PHP package workflow pipelines.

And I also think above static code analysis tools can be added on `grumphp.yml`.

What do you think about these?

**Comment by kpicaza** [comment]
Yes;-D, adding both commands inside the composer `check-all` script will work. 
The check-all script is the action configured in Grumphp. 

it will be something like:
```php
    "scripts": {
        "check-all": [
            "@cs-check",
            "@inspect",
+           "@psalm",
            "@test"
        ],
        "test": "phpunit",
        "cs-check": "phpcs src tests",
        "cs-fix": "phpcbf src tests",
        "inspect": "phpstan analyse src -l7 --ansi",
+       "psalm": "psalm"
    },
```
